registry/download.go

```diff
@@ -30,10 +30,17 @@ func DownloadPluginFromGithub(ctx context.Context, localPath string, org string,
 	downloadDir := filepath.Dir(localPath)
 	pluginZipPath := localPath + ".zip"
 	// https://github.com/cloudquery/cloudquery/releases/download/plugins-source-test-v1.1.5/test_darwin_amd64.zip
-	downloadURL := fmt.Sprintf("https://github.com/cloudquery/cloudquery/releases/download/plugins-%s-%s-%s/%s_%s_%s.zip", typ, name, version, name, runtime.GOOS, runtime.GOARCH)
-	if org != "cloudquery" {
-		// https://github.com/yevgenypats/cq-source-test/releases/download/v1.0.1/cq-source-test_darwin_amd64.zip
-		downloadURL = fmt.Sprintf("https://github.com/%s/cq-%s-%s/releases/download/%s/cq-%s-%s_%s_%s.zip", org, typ, name, version, typ, name, runtime.GOOS, runtime.GOARCH)
+	urls := []string{
+		// community plugin format
+		fmt.Sprintf("https://github.com/%s/cq-%s-%s/releases/download/%s/cq-%s-%s_%s_%s.zip", org, typ, name, version, typ, name, runtime.GOOS, runtime.GOARCH),
+	}
+	if org == "cloudquery" {
+		urls = []string{
+			// CloudQuery monorepo plugin
+			fmt.Sprintf("https://github.com/cloudquery/cloudquery/releases/download/plugins-%s-%s-%s/%s_%s_%s.zip", typ, name, version, name, runtime.GOOS, runtime.GOARCH),
+			// fall back to community plugin format if the plugin is not found in the monorepo
+			urls[0],
```

Comment:
why not just `urls = append([]string{...}, urls...)`? urls[0] would fail if we introduce something that leaves the community plugin empty, or would miss things if we come up with more than one format in the future etc. Using `append` would make it resilient to those changes.

Reply:
I felt like this was more readable and explicit, and I would hope that someone changing the code in the future would read the one or two lines of surrounding context before doing so, and we can't really know what those future changes will be. But I don't think it matters, I will change it.

Reply:
append reads more natural to me than a specific index of a pre-set slice etc. 👍